extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -112,9 +114,19 @@ describe('amp-a4a', () => {
     return new TextEncoder('utf-8').encode(str);
   }
 
+  function createA4aElement(doc) {
+    const element = doc.createElement('amp-a4a');
+    element.getAmpDoc = () => {
+      const ampdocService = ampdocServiceFor(doc.defaultView);
+      return ampdocService.getAmpDoc(element);
+    };
+    doc.body.appendChild(element);
```

Comment:
This is tested inside of a hermetic doc. The whole document is thrown out so this is ok.